schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/state/details/InitializedStateDetails.java

```diff
@@ -25,50 +25,51 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class InitializedStateDetails implements AbstractStateDetails {
 
-    public class MergedBranchDetails {
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class MergeInfo {
```

Comment:
This is not really part of `InitializedStateDetails`, better to move out from this class.